### ISSUE TITLE
st-util gives error 3

### ISSUE THREAD

**Issue by emmettammons** [texane/stlink#231]
i just installed st-util on my ubuntu 12.04 system.  i must have botched it somehow because when i plug in my nucleo-f103rb ubuntu recognizes it and opens it as a file system.   but when i do st-util from the command line i get error 3 can't open device.  before i run st-util  i change the environmental variable STLINK_DEVICE to reflect the bus:device.  i'm thinking that the best thing to do is dump the entire install and start over but i'm not sure how to back everything out.
any help would be appreciated

**Comment by xor-gate** [comment]
This is an old issue, please test against latest master code of stlink tools. Reopen a new issue when similar problem still exists.

**Comment by Nightwalker-87** [comment]
Any hidden bugs left here with this board type?

**Comment by Nightwalker-87** [comment]
@emmettammons: I am currently reviewing old leftovers.
Is this resolved with Release v1.2.0? Please verify.

**Comment by emmettammons** [comment]
i've been away from computer programming for several years now.  i
don't remember what this issue was.

thanks.

emmett

**Comment by Nightwalker-87** [comment]
@emmettammons: Thanks for your feedback. I understand, as this has been left untouched for such a long time. I'm sorry to see that. :-/ 
I've just started to take action as one of the new project contributors.
Closing this now as resolved.

**Comment by Nightwalker-87** [comment]
@emmettammons: Thanks for your feedback. I understand as this has been left untouched for such a long time. I'm sorry to see that. :-/ 
I've just started to take action as one of the new project contributors.
Closing this now as resolved.
